src/Surfnet/ServiceProviderDashboard/Infrastructure/Jira/Repository/DevelopmentIssueRepository.php

```diff
@@ -49,6 +49,12 @@ public function shouldFailCreateIssue(): void
         $this->failIssueCreation = true;
         $this->storeData();
     }
+    public function shouldNotFailCreateIssue(): void
```

Comment:
This is a weird method name. No method should ever fail, but they do... not clear to me

Reply:
Haha, this a method that modifies the behaviour of a fixture. You can enable it to simulate Jira being down.

Reply:
Why not call it enableJiraDownSimulation() then?

Reply:
I find this more fitting in functional tests, but my pardigm is mainly gherkin oriented.